client/js/screens/AuthLoading/AuthLoadingContainer.js

```diff
@@ -9,7 +9,9 @@ class AuthLoadingContainer extends Component {
 
   _bootstrapAsync = async () => {
     const userToken = await getViewer();
-    this.props.navigation.navigate(userToken ? 'App' : 'Auth');
+    this.props.navigation.navigate(userToken ? 'App' : 'Auth', {
+      userToken: JSON.parse(userToken),
```

Comment:
if your userToken is stored in the AsyncStorage than you don't need to pass it as param to App or Auth screen.

Reply:
It is used to get the userToken with `compose()` function in a recursive way on `HomeContainer.js`